### PR TITLE
Replace bower rails for webpacker

### LINKED ISSUE #25: Replace bower-rails with webpacker

### PR DESCRIPTION
**Pull request by dvdgs88**
Closes #25

### CODE REVIEW

File: pathfinder.rb [hunk 1]
```diff
@@ -60,8 +61,11 @@ def ask_for_recipes
      add_recipe(Recipes::Sidekiq.new(self))
      add_recipe(Recipes::SimpleForm.new(self))
      add_recipe(Recipes::Status.new(self))
-     add_recipe(Recipes::BowerRails.new(self))
-     add_recipe(Recipes::ActiveAdmin.new(self))
+     add_recipe(Recipes::Webpacker.new(self))
+     if @template.yes?('Do you want to use Modernizr?')
+       add_recipe(Recipes::Modernizr.new(self))
+     end
+     add_recipe(Recipes::ActiveAdmin.new(´self))
```
Comment (pablorc) [inline pathfinder.rb]: A wild ´ appeared!

File: recipes/webpacker.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+module Recipes
+  class Webpacker < Base
+
+    def gems
+      @template.gem 'webpacker', '~> 3.0'
+    end
+
+    def init_file
+      @template.run 'rails webpacker:install'
+      @template.run "yarn add #{package_command_resources}"
```
Comment (pablorc) [inline recipes/webpacker.rb]: I don't have yarn installed. Maybe we should install it before in the script, or document it at the README?

**Review by pablorc — CHANGES_REQUESTED** [review]
Small fixes requested.

**Comment by dvdgs88** [comment]
@pablorc @xredo I've added the official installation instructions link to the readme, as the installation is specific to the user OS.

**Pull request merged**